internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/event.go

```diff
@@ -167,7 +167,7 @@ func SendEvent(
 		Message:     msg(err),
 		Type:        "info",
 		Duration:    client.DurationSeconds(duration),
-		Results: map[string]any{
+		Params: map[string]any{
```

Comment:
Should be Params instead of Results based on KBC team feedback: https://github.com/keboola/event-schema/pull/22#discussion_r1824305903

Reply:
I commented it in the PR. Please bare in mind, the `telemetry` team scrapes the events and take the `results` currently. You have to align with them. Also the Streams telemetry is currently binded to `results` to be correctly billed.